src/main/java/io/github/dsse/helpers/SimpleECDSAVerifier.java

```diff
@@ -0,0 +1,35 @@
+package io.github.dsse.helpers;
+
+import io.github.dsse.models.Verifier;
+import java.security.InvalidKeyException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.Signature;
+import java.security.SignatureException;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+
+public class SimpleECDSAVerifier implements Verifier {
+
+  private PublicKey publicKey;
+
+  @Override
+  public boolean verify(byte[] publicKeyByteArray, byte[] encryptedMessage, String message)
+      throws NoSuchAlgorithmException, SignatureException, InvalidKeySpecException,
+          InvalidKeyException {
+    Signature signature = Signature.getInstance("SHA1withECDSA");
+    // Create the public key from the byte array
+    PublicKey publicKey =
+        KeyFactory.getInstance("ECDSA").generatePublic(new X509EncodedKeySpec(publicKeyByteArray));
+    this.publicKey = publicKey;
+    signature.initVerify(publicKey);
+    signature.update(message.getBytes());
+    return signature.verify(encryptedMessage);
+  }
+
+  @Override
+  public String getKeyId() {
```

Comment:
FWIW the keyId isn't usually the public key itself but rather a user-friendly string that can be used to identify it.

Would it make sense to let people set this themselves?  (I actually don't see a constructor which is a bit confusing...)

Reply:
For 'SimpleECDSAVerifier' I'd just have it be a member variable and populate it when you populate publicKey.  E.g. have the user do it.

If we had a KMS signer it would probably just take the keyid/name and use it to interface with KMS (either getting the key or asking the KMS to do the verification)

Reply:
I can add that, but it will not match the behavior shown here:

https://github.com/secure-systems-lab/dsse/blob/master/implementation/ecdsa.py#L33

Is this OK?